src/testing/README.md

```diff
@@ -320,7 +320,12 @@ Here we're using the `setChildren()` api on the baseAssertion, and we're using t
 Assertion Template has the following api's:
 
 ```
+insertBefore(selector: string, children: DNode[]): AssertionTemplateResult;
```

Comment:
With the convenience APIs, do you think that we should make the naming more specific to avoid any possible confusion on usage? i.e. `appendChildren`, `prependChildren`, `replaceChildren`, `insertChildrenAfter` and `insertChildrenBefore`?

Reply:
I think at that point you don't gain much over the `setChildren` and `insertChildren` api's. I figured it seemed obvious enough it was operating with nodes and I guess the signature of taking an array of `DNodes[]`, probably covers it? I'm happy to close this PR if we think thats not the case though?

Reply:
I probably need to adjust the `insertChildren` api too anyway, as it doesn't really insert children, it inserts siblings...

Reply:
That's true re `insertChildren`. I see your point about the signature helping to avoid confusion for their purpose, it was just that all the existing APIs were explicit so was caught a bit off guard with additional APIs without the `property`/`children` suffix.

Reply:
I think the only reason the other api's were/are explicit is because `set` clearly does conflict across the 2, i'm not sure the ones i've added here really do (perhaps `replace` at worst does?).

Reply:
But also true, at that point they don't really add much value over the existing APIs.

Reply:
Let's get this in, I think you're right that the naming and then the signature should be clear enough about the APIs usage. I do think that they offer a nice convenience over using the more verbose `setChildren` and `insertSibling` API.